system-requirements.md

```diff
@@ -0,0 +1,42 @@
+# OpenSAFELY Backend System Requirements
+
+## Base System
+
+The OpenSAFELY backend is designed and tested to run on an Ubuntu 20.04 LTS VM.
+
+This should have access to an appropriate Ubuntu archive mirror, and be
+configured to apply security updates automatically.
+
+
+## Dependencies
+
+The required packages are `python3` (which is python 3.8 in 20.04) and
+`docker.io` (the Ubuntu packaged version of docker).
+
+The explicit list can be found in [core-packages.txt](core-packages.txt).
+
+
+## Services
+
+The is currently a single service, `job-runner`, which is installed in /srv/jobrunner.
+A `jobrunner` user is created to run it, and appropriate systemd unit file is installed.
+
+This service does not listen on any port, but regularly polls out to the
+OpenSAFELY platform, for new jobs or publishing job status or results.
```

Comment:
I know what this means, but it could be a little clearer I think.  In particular, I don't think "polls out" is the right term for for publishing job statuses etc.

How about:

> ...but regularly initiates requests to the OpenSAFELY job server to request new jobs and to publish job statuses and results.

Reply:
Should we also mention here that it makes requests for docker images and git repos?

Reply:
yes, and yes, thanks